specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-03-02-preview/managedClusters.json

```diff
@@ -6233,12 +6233,12 @@
       "description": "Only apply to AAD clusters, specifies the format of returned kubeconfig. Format 'azure' will return azure auth-provider kubeconfig; format 'exec' will return exec format kubeconfig, which requires kubelogin binary in the path.",
       "x-ms-parameter-location": "method"
     },
-    "IgnorePodDisruptionBudgetParameter": {
-      "name": "ignore-pod-disruption-budget",
+    "NoPodDisruptionBudgetParameter": {
```

Comment:
@tonche: Just because az cli has a restriction doesn't mean that our REST API should be subject to that restriction. `IgnorePodDisruptionBudget` was the name agreed upon by folks when discussed and IMO we should keep using that name.